html5lib/tests/tree_construction.py

```diff
@@ -51,14 +51,17 @@ def runtest(self):
         fragmentContainer = self.test['document-fragment']
         expected = self.test['document']
         expectedErrors = self.test['errors'].split("\n") if self.test['errors'] else []
+        script = True
+        if 'script-off' in self.test:
+            script = False
```

Comment:
>tree-construction/scripted/webkit01.dat:1<script>document.write("2")</script>3
Huh. Is that supposed to pass? Tests under tree-construction/scripted/ contain neither ```#script-on``` nor ```#script-off```. No idea here.